package.json

```diff
@@ -27,6 +28,7 @@
     "request-promise": "^4.2.4"
   },
   "devDependencies": {
-    "eslint": "7.7.0"
+    "eslint": "7.7.0",
+    "jest": "^26.6.3"
```

Comment:
Added prettier as well, `"prettier": "2.0.5"`

Reply:
@MohdRafiUddin Will open a separate PR for lint/prettifier as it will cause changes in almost all files